docs/install.md

```diff
@@ -12,6 +12,15 @@ sudo apt install python3 python3-pip libusb-1.0-0
 
 *libusb-1.0-0* is not required if you intend to only use `mvt-ios` and not `mvt-android`.
 
+### Note: For Arch Linux
+
+Use the following command for installing the tools:
+
+```bash
+sudo pacman -Syyy
```

Comment:
One `y` is enough, two is overkill, but three? I don't think it has any effect.

Reply:
I see. My bad, I'll correct it to one y. Thank you!